### DISCUSSION TITLE
Límite de entrega, pokedex :(

### DISCUSSION THREAD

**Discussion by AlexAntonioG** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#490] [Q&A] [answered]
Solo quería preguntar acerca de cuando es la fecha límite de pokedex ?, en las publicaciones ví que era el 20 , pero recuerdo a ver visto o escuchado que es hasta el 27.

No me ha dado tiempo por la escuela y así, aun hay tiempo? :(

**Comment by OscarPedroza** [comment, marked as answer]
Así es compañero, la nueva fecha de entrega es el 27 de marzo, aun hay tiempo.

**Reply by AlexAntonioG** [reply]
Muchas gracias 🤗

**Comment by RodolfoMorquecho** [comment]
Si, la recorrieron para el 27 aunque son fechas mas estipuladas para que no te atrases que como limite de que no las puedas enviar mas.

**Reply by AlexAntonioG** [reply]
Gracias 🤗
